Source/DafnyCore/Compilers/ResolvedDesugaredExecutableDafny/Dafny-compiler-fdafny.dfy

```diff
@@ -2,15 +2,209 @@ module {:extern "ResolvedDesugaredExecutableDafnyPlugin"} ResolvedDesugaredExecu
   import opened DAST
   import PrettyPrinter
   import UnsupportedFeature
+  import DAM
 
   class COMP {
```

Comment:
Class `COMP` only has static members. Remove the class altogether and instead declare the members as module-level members.

Reply:
I think this should be fine as long as `Compiler-dafny.cs` does not depend on the fact that `COMP` is a class by way of its translation to C# in `GeneratedFromDafny.cs`.